src/rules.ts

```diff
@@ -593,7 +600,9 @@ export const rules: Rule[] = [
       ],
```

Comment:
can you add an example showing what the new options do?

Reply:
Example added